src/SFA.DAS.QnA.Application/Commands/Files/UploadFile/SubmitPageOfFilesHandler.cs

```diff
@@ -63,7 +63,7 @@ private HandlerResponse<SetPageAnswersResponse> ValidateRequest(SubmitPageOfFile
             {
                 return new HandlerResponse<SetPageAnswersResponse>(success: false, message: "Cannot find requested page.");
             }
-            else if (request.Files is null || !request.Files.Any())
```

Comment:
any reason to take this out? Seems fine to leave in to me.  Your choice.

Reply:
.Any is an incorrect assumption as it takes presence over any Required Validators on each question, thus giving back the wrong error message.